spec/stepmod/concept_spec.rb

```diff
@@ -85,9 +87,11 @@
         // STEPmod path:
         === complement
 
-        domain:[ISO 10303 resource]
+        domain:[resource: action_schema]
+
+        entity data type that represents a/an complement entity
 
-        A *complement* is a relationship that is between
+        NOTE: A *complement* is a relationship that is between
```

Comment:
This NOTE is a long one that contains blank lines. Blank lines causes Metanorma to split text into multiple paragraphs. In this case, we need to use an alternative syntax of NOTE.

The following three forms are equivalent:

Single paragraph:
```adoc
NOTE: This is a note.
```

Continuation style:
```adoc
NOTE: This is a note.
+
* list in a note.
```

Block style:
```adoc
[NOTE]
--
This is a note.

* list in a note.
--
```

We need to use the block-style syntax (the third one).